api/index.php

```diff
@@ -128,29 +128,42 @@ function decode_json_throw_errors($inputString) {
 });
 
 ####################### ASSIGNMENTS ###########################3
-$app->get('/assignments/get/all', function (Request $request, Response $response) use ($dataMgr){
+$app->get('/assignment/get', function (Request $request, Response $response) use ($dataMgr){
 
 
     $params = json_decode($request->getBody(), true);
     $dataMgr->setCourseFromID(new CourseID($params['courseID']));
-    $assignments = $dataMgr->getAssignments();
-    $newResponse = $response->withJson($assignments);
-    return $newResponse;
+    
+    if ($params["assignmentIDs"] == "all"){
```

Comment:
when you make the schema, this may have to change. just something to keep in mind.